tmt/steps/execute/__init__.py

```diff
@@ -854,7 +786,7 @@ def extract_tmt_report_results(self, invocation: TestInvocation) -> list["tmt.Re
 
         collection.validate()
 
-        return self._process_results_reduce(invocation, collection.results)
+        return [tmt.Result.from_serialized(r) for r in collection.results]
```

Comment:
Check the docstring of this method, there's one line that no longer applies.

Reply:
Thanks, I've just removed the line.